### PR TITLE
CI: add ppc64le arch in the CI

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by nplanel** [comment]
run skydive-scale-tests

**Comment by nplanel** [comment]
run skydive-ppc64le-tests

**Comment by nplanel** [comment]
run skydive-ppc64le-tests

**Comment by nplanel** [comment]
run skydive-ppc64le-tests

File: tests/lxd_test.go [hunk 1]
```diff
@@ -24,13 +24,18 @@ package tests
...
 	"fmt"
+	"os"
 	"testing"
 
 	g "github.com/skydive-project/skydive/gremlin"
 	"github.com/skydive-project/skydive/tests/helper"
 )
 
 func TestLxdSimple(t *testing.T) {
+	if os.Getenv("GOARCH") == "ppc64le" {
```
Comment (lebauce) [inline tests/lxd_test.go]: if runtime.GOARCH == "ppc64le" {

**Comment by lebauce** [comment]
It seems `TestFlowsEBPF` also has to be skipped for now

**Comment by lebauce** [comment]
run skydive-ppc64le-tests

**Pull request merged**